cmd/minikube/cmd/root.go

```diff
@@ -115,8 +118,11 @@ func setFlagsUsingViper() {
 
 func init() {
 	RootCmd.PersistentFlags().Bool(showLibmachineLogs, false, "Show logs from libmachine.")
-	RootCmd.PersistentFlags().String(username, "", "User name for the virtual machine.")
-	RootCmd.PersistentFlags().String(password, "", "Password for the virtual machine.")
+	RootCmd.PersistentFlags().String(username, "", "User name for the virtual machine registration.")
+	RootCmd.PersistentFlags().String(password, "", "Password for the virtual machine registration.")
+	RootCmd.PersistentFlags().String(httpProxy, "", "HTTP proxy for virtual machine.")
+	RootCmd.PersistentFlags().String(httpsProxy, "", "HTTPS proxy for virtual machine.")
```

Comment:
Forget this one, I was thinking wrong. I think I understand now. Tools needs to select the proxy depending on what they want to proxy, right?